src/main/java/si/slotex/nlp/gui/models/Sentence.java

```diff
@@ -0,0 +1,42 @@
+package si.slotex.nlp.gui.models;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class Sentence implements Serializable
```

Comment:
Do we really need to define new entities since we've already got them in our slotex-nlp-entity package.

https://mvnrepository.com/artifact/si.slotex.nlp/slotex-nlp-entity/1.0.0

* Sentence
* DocTag
* Token...